rules/mv.go

```diff
@@ -0,0 +1,48 @@
+package rules
+
+import (
+	"github.com/jessevdk/go-flags"
+	"github.com/nopecmd/nope/match"
+	"github.com/nopecmd/nope/models"
+	"github.com/nopecmd/nope/shells"
+	"strings"
+)
+
+var mvFlags struct {
+	Interactive bool `short:"i"`
+	Force       bool `short:"f"`
+}
+
+const (
+	mvBaseCommand = "mv"
+)
+
+func isMatchMv(cmd models.Command) bool {
+	return cmd.BaseCommand == mvBaseCommand
+}
+
+func getUndoMv(cmd models.Command) string {
+	// for now, just remove all flags
+	var err error
+	cmd.TokensWithoutFlags, err = flags.ParseArgs(&mvFlags, cmd.Tokens[1:])
+
+	if err != nil {
+		panic(err)
+	}
+
```

Comment:
I think the logic above should be moved here: https://github.com/nopecmd/nope/blob/master/match/match.go#L18, I really feel that setting the flags for the command (or setting anything for the command) shouldn't be in the scope of getUndo*. It should receive a fully featured command object, and get an undo action according to that. It really shouldn't be doing any initial parsing of the command (such as settings flags).

We can add a `flags` field to Rule model and just pass the flags to this function: https://github.com/nopecmd/nope/blob/master/match/match.go#L10